testsuite/tests/shape-index/index_aliases.reference

```diff
@@ -21,13 +32,22 @@ Resolved: Index_aliases.1 :
   A (File "index_aliases.ml", line 19, characters 11-12)
 
 Uid of decls:
+Index_aliases.12: M (File "index_aliases.ml", line 31, characters 27-28)
 Index_aliases.1: A (File "index_aliases.ml", line 18, characters 7-8)
 Index_aliases.2: B (File "index_aliases.ml", line 19, characters 7-8)
+Index_aliases.18: Y (File "index_aliases.ml", line 34, characters 7-8)
+Index_aliases.17: X (File "index_aliases.ml", line 33, characters 7-8)
 Index_aliases.5: F (File "index_aliases.ml", line 21, characters 7-8)
 Index_aliases.7: C (File "index_aliases.ml", line 23, characters 7-8)
+Index_aliases.13: S (File "index_aliases.ml", line 31, characters 12-13)
 Index_aliases.9: D (File "index_aliases.ml", line 26, characters 7-8)
+Index_aliases.15: s (File "index_aliases.ml", line 32, characters 45-46)
+Index_aliases.19: Z (File "index_aliases.ml", line 35, characters 7-8)
 Index_aliases.8: C' (File "index_aliases.ml", line 25, characters 7-9)
+Index_aliases.14: x (File "index_aliases.ml", line 32, characters 4-5)
 Index_aliases.10: G (File "index_aliases.ml", line 28, characters 7-8)
+Index_aliases.16: M (File "index_aliases.ml", line 32, characters 30-31)
 Index_aliases.6: F' (File "index_aliases.ml", line 22, characters 7-9)
 Index_aliases.3: t (File "index_aliases.ml", line 21, characters 23-24)
+Index_aliases.11: s (File "index_aliases.ml", line 31, characters 39-40)
 Index_aliases.0: t (File "index_aliases.ml", line 18, characters 23-24)
```

Comment:
There are many more names listed here now, do you agree that this is a good change? (Or is it a regression in some respect?)

Reply:
That's expected, this list reflects the fact that I added new definitions in the module. The ordering is surprising though, but no regression here.